pkg/kubefed2/unjoin.go

```diff
@@ -241,25 +191,24 @@ func performPreflightChecksForUnjoin(clusterClientset client.Interface, name, ho
 // removeFromClusterRegistry handles removing the cluster from the cluster registry and
 // reports progress.
 func removeFromClusterRegistry(hostConfig *rest.Config, clusterNamespace, host, unjoiningClusterName string,
-	dryRun bool) error {
+	dryRun bool) {
 
 	// Get the cluster registry clientset using the host cluster config.
 	crClientset, err := util.ClusterRegistryClientset(hostConfig)
 	if err != nil {
 		glog.V(2).Infof("Failed to get cluster registry clientset: %v", err)
```

Comment:
Given that the error is no longer returned to be logged by the caller, this log call should be `glog.Errorf` instead of `Infof`.